libsawtooth/RELEASE_NOTES.md

```diff
@@ -1,5 +1,18 @@
 # Release Notes
 
+## Changes in libsawtooth 0.3.1
+
+* Move Sawtooth validator Rust code that could be easily separated from the
+  Python code into libsawtooth
+* Add feature guard to prevent unused import warning
+* Add protobuf generation to libsawtooth
+* Add stable, experimental features to all crates
+* Replace libsawtooth metrics with metrics crate
+* Use libsawtooth's metrics mod
+* Create traits for metrics module
```

Comment:
These can probably just be removed; all of the things that use metrics in libsawtooth are brand new to libsawtooth (moved from validator), so we don't really need to say any of this.